### ISSUE TITLE
Provide a valid .htmllintrc file with options

### ISSUE THREAD

**Issue by mikebell** [htmllint/htmllint-cli#11]
I'm trying to get a project up and running but I need to disable some rules so they don't get triggered on CI run.

My config is:

```
{
    // names of npm modules to load into htmllint
    "plugins": [],

    // TODO: export htmllint's default cfg
    "options": [
      {"attr-name-style":"none"},
      {"attr-req-value":"none"}
    ]
}
```

But it doesn't seem to be being used. What is the correct way of using htmllint options in the rc file?

**Comment by coalman** [comment]
Try specifying your options like this:

```
{
    // names of npm modules to load into htmllint
    "plugins": [],

    // TODO: export htmllint's default cfg
    "attr-name-style": false,
    "attr-req-value": false
}
```

**Comment by mikebell** [comment]
Thanks this works great.
